cluster/charts/crossplane/README.md

```diff
@@ -1,3 +1,9 @@
+---
```

Comment:
I believe that this front matter is the content that is supposed to be excluded, so this should not be re-added back in with this commit or with running `make reviewable` in general, right?

Reply:
ahh did not realize that had happened, one moment